extensions/oauth/src/test/resources/authorization-code/authorization-code-localcallbackref-config.xml

```diff
@@ -14,22 +14,21 @@
 
     <oauth:token-manager-config name="tokenManagerConfig"/>
 
+    <oauth:authorization-code-grant-type
```

Comment:
Do we still support this config inside http:request-connection?. And if so, why do you had to move it to a top level config?